.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: Continuous Integration
+on:
+  push:
+    branches: [master]
+  pull_request:
+jobs:
+  ruby:
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [2.4, 2.5, 2.6]
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+    - name: Setup Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+    - name: Install dependencies
+      run: bundle install
+    - name: Run rspec
+      run: bundle exec rspec
+    - name: Run rubocop
+      run: bundle exec rubocop --display-cop-names
+    - name: Run integration testing
```

Comment:
up to you, but you may want to have unittest, integration tests, and lint be separate jobs

Reply:
I didn't do this for now because I didn't like combinatorial explosion (as we test for multiple ruby versions).